packages/artifactor/test/contracts.js

```diff
@@ -96,7 +95,7 @@ describe("artifactor + require", () => {
   });
 
   before(() =>
-    web3.eth.getAccounts().then(_accounts => {
+    interfaceAdapter.eth.getAccounts().then(_accounts => {
```

Comment:
Ah, this sort of thing is the culprit! If you want to do this, you should make a `getAccounts` method for real on the interface adapter

Reply:
Ahh, so we should have started with `setProvider` then 🤦‍♂ .